### ISSUE TITLE
Clean up browser logs

### ISSUE THREAD

**Issue by d-ivashchuk** [lost-pixel/lost-pixel#154]
### Feature description

Potentially introduce the flag to mute the browser logs.

**Comment by narutosstudent** [comment]
@d-ivashchuk maybe i could start with this?

forking the repo right now

do you've a discord i can add you in?

**Comment by d-ivashchuk** [comment]
This one seems to be perfect 👍🏼 @narutosstudent

**Comment by d-ivashchuk** [comment]
@narutosstudent if you don't mind I will point your to the rights parts of codebase so you have easier time navigating! But tomorrow already, quite a day it was!

**Comment by narutosstudent** [comment]
@d-ivashchuk sounds good, discord shouldn't be necessary tbh :joy: 

Github got enough ways to communicate, even emojis here lol :skull: 

I'm happy to also hack on features eventually, will go through the codebase this evening and make sure to study it a bit, will try to take notes of questions I've that you could respond to.

Thank you :heart: .

**Comment by narutosstudent** [comment]
@d-ivashchuk It might be a bit small, but check the top left chrome group of mine, lost-pixel :sunglasses: :fire: got the fork and this github issue open, ready anytime :muscle: :smile: 

![Screenshot from 2022-09-01 08-53-16](https://user-images.githubusercontent.com/49603590/187850899-e6fcb225-6430-4096-9e46-2b620c27b029.png)

**Comment by d-ivashchuk** [comment]
If you will have time and gather some amount of questions already - we can even jump on quick call in the evening 😊

**Comment by judicaelandria** [comment]
> If you will have time and gather some amount of questions already - we can even jump on quick call in the evening 😊

Hey can I join too? would love to know more about this project for future contribution :)

**Comment by d-ivashchuk** [comment]
@judicaelandria absolutely, I will work on the weekend to create some issues that would be good first issues and make it easier for people of various knowledge and backgrounds to contribute! Thanks for your interest 🥇

**Comment by d-ivashchuk** [comment]
@narutosstudent soo the scope of the issue is quite limited. Trying to be more verbose in debugging at some point we've added [this chunk of code](https://github.com/lost-pixel/lost-pixel/blob/0653f34da74dc066fb6d42afab8cdd862bbafcc4/src/shots/shots.ts#L26-L40) which is responsible for piping the browser logs to the general logs of lost-pixel. Those are seen on the runs in the CI and if you have some problems with your screenshots it makes it easier to debug. 
![image](https://user-images.githubusercontent.com/29632358/187884682-5bb4b731-36ac-4292-80c4-8c63f1110e28.png)

the main problem with it is that sometimes it just pollutes the console in cases where we are not interested in the logs. Meaning we have couple of options to remedy this:

1.  introduce `{muteBrowserLogs: true}` logs flag to `lostpixel.config`
2. introduce general logs flag `{logs: 'VERBOSE' | 'INFO' }` - `INFO` being the default and just having all the informational and error logs from lost-pixel run and `VERBOSE` covering the case for browser logs. the question here would be if we would not need to introduce other log levels in the future. this approach implies we'd at least try to make it right from first shot

I am very much happy to listen to your suggestions and explore good practices though @narutosstudent, I bet tools like `Cypress` or `Playwright` have it figured out already and maybe you could check those out first :D 

@chriskalmar thoughts?

**Comment by judicaelandria** [comment]
> @judicaelandria absolutely, I will work on the weekend to create some issues that would be good first issues and make it easier for people of various knowledge and backgrounds to contribute! Thanks for your interest 🥇

cool! here's my email for the invitation judicaelandria7@gmail.com

**Comment by narutosstudent** [comment]
@d-ivashchuk Sorry for the late response, was busy :smile: :heart:

Let me read your reply.

Yes, I'm happy to jump on a call.

Feel free to invite me to a meeting that fits for you, Zoom or Google Meet :heart:

My email: tigerabrodi@gmail.com 

Though we may not need a meeting yet, I'm happy to pair code and learn more about the codebase :heart: 

@d-ivashchuk At work, we're using this package to make sure when a console error happens, that the test fails: https://www.npmjs.com/package/cypress-fail-on-console-error

> the main problem with it is that sometimes it just pollutes the console in cases where we are not interested in the logs. Meaning we have couple of options to remedy this:

I see, so the goal here is for the logs in CI to be cleaned up, so it doesn't get too _dirty_, am I right?

**Comment by narutosstudent** [comment]
@judicaelandria You can continue on this if you want, I've decided to step away from open-source as I'm busy with other things. :+1:
